src/Serilog.Sinks.ElmahIO/LoggerConfigurationElmahIOExtensions.cs

```diff
@@ -36,7 +36,7 @@ public static class LoggerConfigurationElmahIOExtensions
         public static LoggerConfiguration ElmahIO(
             this LoggerSinkConfiguration loggerConfiguration,
              Guid logId,
-            LogEventLevel restrictedToMinimumLevel = LogEventLevel.Error,
+            LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
```

Comment:
Understandable change as you now process more than errors only, but it is a bit hidden away as default value and can be considered breaking. Maybe good to document it somewhere?